Index: debian/src/python/CMakeLists.txt
===================================================================
--- debian.orig/src/python/CMakeLists.txt	2016-10-25 13:10:05.035463950 +0200
+++ debian/src/python/CMakeLists.txt	2016-10-25 13:10:41.128428516 +0200
@@ -13,6 +13,7 @@
     FILES_MATCHING PATTERN *.py
     # Avoid empty directory
     PATTERN "bruker" EXCLUDE
+    PATTERN "nifti" EXCLUDE
 )
 
 file(GLOB_RECURSE python_files "*.py")
