Index: debian/CMakeLists.txt
===================================================================
--- debian.orig/CMakeLists.txt	2016-10-25 14:11:33.246029475 +0200
+++ debian/CMakeLists.txt	2016-10-25 14:14:05.118088177 +0200
@@ -1,4 +1,10 @@
 project(dicomifier)
+set(dicomifier_MAJOR_VERSION 0)
+set(dicomifier_MINOR_VERSION 10)
+set(dicomifier_PATCH_VERSION 3)
+set(dicomifier_VERSION
+    ${dicomifier_MAJOR_VERSION}.${dicomifier_MINOR_VERSION}.${dicomifier_PATCH_VERSION})
+
 cmake_minimum_required(VERSION 2.6.0)
 
 option(BUILD_EXAMPLES "Build the examples directory." ON)
Index: debian/src/lib/CMakeLists.txt
===================================================================
--- debian.orig/src/lib/CMakeLists.txt	2016-08-23 11:25:09.762869005 +0200
+++ debian/src/lib/CMakeLists.txt	2016-10-25 14:13:01.640391769 +0200
@@ -25,7 +25,10 @@
     ${Log4Cpp_LIBRARIES})
 
 set_target_properties(
-    libdicomifier PROPERTIES OUTPUT_NAME dicomifier)
+    libdicomifier PROPERTIES 
+    OUTPUT_NAME dicomifier
+    VERSION ${dicomifier_VERSION}
+    SOVERSION ${dicomifier_MAJOR_VERSION})
 
 install(
     FILES ${headers} ${templates} DESTINATION include/dicomifier)
